src/renderer/resources/style/components/_errors.scss

```diff
@@ -0,0 +1,21 @@
+.errors {
+    color: firebrick;
+    border: solid 1px firebrick;
+    background: rgba( red, 0.2 );
```

Comment:
Nitpick, but would you be able to remove the extra spaces between the braces?
See [_context-menu.scss](https://github.com/james-proxy/james/blob/3dae5adc4bc1afe5cf7f8fa5a79b15317f42799a/src/renderer/resources/style/components/_context-menu.scss#L16) for an example of how it looks elsewhere

Reply:
Will do.